src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -60,10 +62,24 @@ bool ScaledDotProductAttentionKey::operator==(const ScaledDotProductAttentionKey
     return retVal;
 }
 
+inline uint8_t* get_attn_mask_ptr(const PlainTensor& attention_mask, size_t b, size_t h) {
+    auto attn_mask_prec = attention_mask.get_precision();
+    uint8_t* attn_mask_ptr = nullptr;
+
+    if (attn_mask_prec == ov::element::f32) {
+        attn_mask_ptr = reinterpret_cast<uint8_t*>(&attention_mask.at<float>({b, h, 0, 0}, true));
+    } else {
+        attn_mask_ptr = reinterpret_cast<uint8_t*>(&attention_mask.at<bfloat16>({b, h, 0, 0}, true));
+    }
+
+    return attn_mask_ptr;
+}
+
 // default implementation: reference
 template <ScaledDotProductAttention::KernelTypes KType, typename T>
 struct MHAKernel {
-    MHAKernel() = default;
```

Comment:
How about `MHAKernel() = delete;` to not allow uninitialized object?

Reply:
Sounds Great! will fix this

Reply:
fix in a518f574611468d8a444da0e635480f2572bb316